### PR TITLE
Fixed the documetation menu

### PR DESCRIPTION
**Pull request by PacoDw**

### CODE REVIEW

File: website/mongodbatlas.erb [hunk 1]
```diff
@@ -136,6 +139,9 @@
                     <li<%= sidebar_current("docs-mongodbatlas-resource-x509-authentication-database-user") %>>
                         <a href="/docs/providers/mongodbatlas/r/x509_authentication_database_user.html">mongodbatlas_x509_authentication_database_user</a>
                     </li>
+                    <li<%= sidebar_current("docs-mongodbatlas-resource-cloud-provider-snapshot-backup-policy") %>>
+                        <a href="/docs/providers/mongodbatlas/r/cloud_provider_snapshot_backup_policy.htmll">mongodbatlas_cloud_provider_snapshot_backup_policy</a>
```
Comment (themantissa) [inline website/mongodbatlas.erb]: html has an extra l, assuming not intentional?

**Review by themantissa — CHANGES_REQUESTED** [review]
Think there is a misspelling to fix first, thanks!

File: website/mongodbatlas.erb (hunk 1 above)
Comment (PacoDw) [reply website/mongodbatlas.erb]: sorry let me fix it

**Review by themantissa — APPROVED** [review]
LGTM!! Thank you :)

**Review by marinsalinas — APPROVED** [review]

**Pull request merged**